### PR TITLE
overview-image-new

### PR DESCRIPTION
**Pull request by philipwinklerdelmic**

### CODE REVIEW

**Comment by philipwinklerdelmic** [comment]
I rebased the commits and deleted the old icons. The changes are the same as in the old pull request.

File: src/odemis/gui/comp/legend.py [hunk 1]
```diff
@@ -58,10 +58,15 @@ def __init__(self, parent, wid=-1, pos=(0, 0), size=wx.DefaultSize,
...
             (stream.ARStream, img.getBitmap("icon/ico_blending_ang.png")),
             (stream.SpectrumStream, img.getBitmap("icon/ico_blending_spec.png")),
-            (stream.OpticalStream, img.getBitmap("icon/ico_blending_opt.png")),
             (stream.CLStream, img.getBitmap("icon/ico_blending_opt.png")),  # same as optical
             (stream.EMStream, img.getBitmap("icon/ico_blending_sem.png")),
             (stream.RGBStream, img.getBitmap("icon/ico_blending_goal.png")),
+            (stream.RGBUpdatableStream, img.getBitmap("icon/ico_blending_map.png")),
+            (stream.RGBCameraStream, img.getBitmap("icon/ico_blending_navcam.png")),
+            (stream.BrightfieldStream, img.getBitmap("icon/ico_blending_navcam.png")),
+            (stream.Static2DStream, img.getBitmap("icon/ico_blending_history.png")),
```
Comment (pieleric) [inline src/odemis/gui/comp/legend.py]: Static2DStream includes many streams, including StaticFluoStream, which should be shown with optical icon.

File: src/odemis/gui/model/__init__.py [hunk 2]
```diff
@@ -1385,4 +1385,4 @@ def __init__(self, name, **kwargs):
         self.interpolate_content.value = False
 
         self.mpp.value = 10e-6
-        self.mpp.range = (10e-6, 1e-3)
+        self.mpp.range = (1e-10, 1e10)  # (10e-6, 1e-3)
```
Comment (pieleric) [inline src/odemis/gui/model/__init__.py]: Why making the maximum mpp so big? That's 10 million kilometers per pixel. We should raise an error if such value happens. Probably 1 is fine as maximum if you want really freedom.

File: src/odemis/gui/comp/overlay/view.py [hunk 3]
```diff
@@ -1417,6 +1421,13 @@ def _draw_rect(ctx, v_center, v_size, colour, alpha):
         ctx.rectangle(x, y, v_size[0], v_size[1])
         ctx.stroke()
 
+    def set_merge_ratio(self, merge_ratio):
+        """
+        Modifies the internal attribute _merge_ratio that controls the transparency 
+        of the history overlay.
+        """
+        self._merge_ratio = merge_ratio
+
```
Comment (pieleric) [inline src/odemis/gui/comp/overlay/view.py]: Add the call to `cnvs.request_drawing_update()` here.

**Comment by philipwinklerdelmic** [comment]
Just changed the mpp in model.__init__ to a max of 1 instead of 1E10 and added a request_drawing_update in the History Overlay as requested.

**Pull request closed without merging**